https://bugs.gentoo.org/898258
https://sourceforge.net/p/plplot/patches/37/

Avoid implicitly declaring exit.  Future compilers will not support
implicit function declarations, so this cmake probe will always fail.

--- a/cmake/modules/TestForHighBitCharacters.c
+++ b/cmake/modules/TestForHighBitCharacters.c
@@ -17,6 +17,6 @@ int
    int i;
    for (i = 0; i < 256; i++)
      if (XOR (islower (i), ISLOWER (i)) || toupper (i) != TOUPPER (i))
-       exit(1);
-   exit (0);
+       return 1;
+   return 0;
 }

